gen2-rgbd-projection/pointcloud_example.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+import cv2
+import numpy as np
+import depthai as dai
+
+try:
+    from projector_3d import PointCloudVisualizer
+except ImportError as e:
+    raise ImportError(f"\033[1;5;31mError occured when importing PCL projector: {e}")
+
+
+# StereoDepth config options. TODO move to command line options
+lrcheck = True  # Better handling for occlusions
+extended = False  # Closer-in minimum depth, disparity range is doubled
+subpixel = False  # Better accuracy for longer distance, fractional disparity 32-levels
+
+
+def create_rgb_pipeline():
+    pipeline = dai.Pipeline()
+
+    # Define sources and outputs
+    camRgb = pipeline.createColorCamera()
+    left = pipeline.createMonoCamera()
+    right = pipeline.createMonoCamera()
+    stereo = pipeline.createStereoDepth()
+    depthOut = pipeline.createXLinkOut()
+    rgbOut = pipeline.createXLinkOut()
+
+    # Properties
+    camRgb.setBoardSocket(dai.CameraBoardSocket.RGB)
+    camRgb.setResolution(dai.ColorCameraProperties.SensorResolution.THE_4_K)
+    camRgb.setIspScale(1, 3)
+    camRgb.setInterleaved(False)
+
+    left.setResolution(dai.MonoCameraProperties.SensorResolution.THE_400_P)
```

Comment:
Why 400_P ?

Reply:
No real reason, I was just messing around with values. Although I've noticed the controls (draggin around the Point Cloud projection) are smoother with the 400_P.

Reply:
Lets place them back to 720 so that pixel to pixel alignment is there.